src/slash-bedrock/share/brl-fetch/distros/fedora

```diff
@@ -53,19 +60,42 @@ EOF
 }
 
 default_release() {
-	list_releases |
+	list_release |
 		sort -n |
 		tail -n1
 }
 
 list_releases() {
+
+	echo
+	echo "Release"
```

Comment:
People script around `brl fetch -R` assuming the outputs can be sanely fed into `brl fetch -r`.  Output comments like this will break such scripting.  Please remove them.

Come Naga we can make a breaking change for this kind of thing.  Feel free to bring it up during Naga's public alpha, once we get there.  In addition to differentiating between stable vs development, we can also highlight things like LTS releases in Ubuntu or somehow indicate what `brl fetch` will default to if left unspecified.

Reply:
I figured that would be the case... I just didn't like the output it was giving without:
```
daniel@danielLaptop ~ [1]> sudo brl fetch fedora_test -R
[sudo] password for daniel: 
[cut here]
27
28
29
30
31
32
33
34
35
rawhide
36
```
If you have any suggestions to improve that (or if you are fine with it) let me know

Reply:
I agree, `rawhide`'s placement is a bit weird.  Try piping `list_release` and `list_development` into `sort`, e.g.

```
list_releases() {
	(
		list_release
		list_development
	) | sort -n
}
```

If you do this, consider removing the `sort -n`s from `list_release` and `list_development` as they would end up being redundant.